app/Domain/Users/Templates/editOwn.blade.php

```diff
@@ -31,57 +31,52 @@
+                                        id="firstname"
                                         class="input"
-                                        {{ session('userdata.isLdap') ? 'disabled' : '' }} 
+                                        :value="$values['firstname']"
+                                        caption="{{ __('label.firstname') }}"
+                                        :disabled="session('userdata.isLdap')"
                                     />
-                                    <br/>
                                 </div>
-                        
+                            
                                 <div class="form-group">
-                                    <x-global::forms.text-input 
-                                        type="text" 
-                                        name="lastname" 
-                                        id="lastname" 
-                                        value="{{ $values['lastname'] }}" 
-                                        labelText="{{ __('label.lastname') }}"
+                                    <x-global::forms.text-input
+                                        name="lastname"
+                                        id="lastname"
                                         class="input"
-                                        {{ session('userdata.isLdap') ? 'disabled' : '' }} 
+                                        :value="$values['lastname']"
+                                        caption="{{ __('label.lastname') }}"
+                                        :disabled="session('userdata.isLdap')"
                                     />
-                                    <br/>
                                 </div>
-                        
+                            
                                 <div class="form-group">
-                                    <x-global::forms.text-input 
-                                        type="text" 
-                                        name="user" 
-                                        id="user" 
-                                        value="{{ $values['user'] }}" 
-                                        labelText="{{ __('label.email') }}"
+                                    <x-global::forms.text-input
+                                        name="user"
+                                        id="user"
                                         class="input"
-                                        {{ session('userdata.isLdap') ? 'disabled' : '' }} 
+                                        :value="$values['user']"
+                                        caption="{{ __('label.email') }}"
+                                        :disabled="session('userdata.isLdap')"
                                     />
-                                    <br/>
                                 </div>
-                        
+                            
                                 <div class="form-group">
```

Comment:
With the components written as is we should be able to remove the surrounding div containers